src/include/basil.h

```diff
@@ -976,3 +977,5 @@ typedef struct basil_response {
 #define BASIL_ERR_TRANSIENT	0x00000001UL
 
 #endif /* _BASIL_H */
+
+// clang-format off
```

Comment:
Shouldn't this be clang-format on?  It is actually needed since it is the end of the file?